py/desispec/util.py

```diff
@@ -85,22 +86,17 @@ def runcmd(cmd, args=None, inputs=[], outputs=[], clobber=False):
 
     #- run command
     if isinstance(cmd, collections.abc.Callable):
+        #- TODO: should we check outputs before returning when cmd is callable?
```

Comment:
It's worth considering whether it makes sense to return the result of callable directly or verify that the desired outputs exist like in the subprocess.call usage.

Reply:
it probably doesn't make sense, but leaving this for a future PR since it's not necessary for deforkification